src/spacel/aws/meta.py

```diff
@@ -17,11 +17,20 @@ def __init__(self):
         self.instance_id = self._get(INSTANCE_ID_URL, DEFAULT_INSTANCE)
         self.az = self._get(PLACEMENT_URL, 'us-east-1a')
         self.region = self.az[:-1]
-        self.user_data = json.loads(self._get(USER_DATA_URL, '{}'))
+        self.user_data = self._parse_user_data(self._get(USER_DATA_URL, '{}'))
         self.name = self.user_data.get('name', 'test')
         self.orbit = self.user_data.get('orbit', 'test')
         self.bastion = self.user_data.get('bastion', False)
 
+    def _parse_user_data(self, data):
+        split = data.splitlines()
+        lines = [line for line in split if line.startswith('#manifest:')]
```

Comment:
Nit: you're collecting all lines that startwith `#manifest:`; but only using the first.

Maybe simple for loop for dat early exit?